### PR TITLE
Task #66301: Core, modules, security update > D 10.3

### PR DESCRIPTION
**Pull request by ew-fedir**

### CODE REVIEW

File: config/default/scheduler.settings.yml [hunk 1]
```diff
@@ -18,7 +18,6 @@ default_unpublish_enable: false
 default_unpublish_required: false
 default_unpublish_revision: false
 hide_seconds: false
-lightweight_cron_access_key: c76c2520945a205e4a2f
```
Comment (ew-fedir) [inline config/default/scheduler.settings.yml:21]: Cron access key now stored in state. 
see `scheduler_update_8209()`

File: config/default/views.settings.yml [hunk 2]
```diff
@@ -2,7 +2,6 @@ _core:
   default_config_hash: uZHsLrDp1ThO0RvupHKcPzLOyVvWexm58JTTHNDo7yc
 display_extenders:
   - ajax_history
-skip_cache: false
```
Comment (ew-fedir) [inline config/default/views.settings.yml:5]: See `views_post_update_remove_skip_cache_setting()`

File: config/default/views.view.facilities_listing_blocks.yml [hunk 3]
```diff
@@ -8,20 +8,6 @@ dependencies:
...
-    - 'taxonomy_term:community_facilities:997c6294-6736-45f8-a800-3d76d82ba733'
-    - 'taxonomy_term:community_facilities:a904338c-8486-4be4-9bbf-fe309f435b71'
-    - 'taxonomy_term:community_facilities:d80f1e60-9ae0-45f7-af12-d52085392cbe'
-    - 'taxonomy_term:community_facilities:ed34145d-4289-4b5e-af9b-53fbb29a0770'
-    - 'taxonomy_term:education_facilities:35ab87dd-20dc-45e7-8783-de22f3071b5d'
-    - 'taxonomy_term:education_facilities:9f45d6d0-3d8f-4076-831c-b33c3ab95ff6'
-    - 'taxonomy_term:education_facilities:d809cada-9cf2-4f68-941a-d208ad128b08'
-    - 'taxonomy_term:health_facilities:6a1b308d-d93e-46e3-8c59-84fdbb89cc59'
-    - 'taxonomy_term:sanitary_facilities:009480af-c5cd-4bd8-9d3f-b19f72426532'
-    - 'taxonomy_term:sanitary_facilities:550dd456-de93-409f-b3ac-f6bf71923a80'
```
Comment (ew-fedir) [inline config/default/views.view.facilities_listing_blocks.yml:24]: I'm not sure about this change. Need farther investigation.

File: config/default/views.view.find_a_campground_or_recreation_site.yml [hunk 4]
```diff
@@ -7,11 +7,6 @@ dependencies:
     - node.type.campground_directory_record
     - taxonomy.vocabulary.region
     - taxonomy.vocabulary.site_type
-  content:
-    - 'taxonomy_term:site_type:464b2caa-cf1d-46da-800f-0cace9d7a7f6'
-    - 'taxonomy_term:site_type:d7b69697-152b-4ef3-b853-2d6ad1340491'
-    - 'taxonomy_term:site_type:f2f63b8d-2f0e-469b-a64b-fe1e0922cdc6'
-    - 'taxonomy_term:site_type:f7783847-bf85-45a1-8f05-21e8240a4759'
```
Comment (ew-fedir) [inline config/default/views.view.find_a_campground_or_recreation_site.yml:14]: I'm not sure about this change. Need farther investigation.

File: config/default/views.view.facilities_listing_blocks.yml (hunk 3 above)
Comment (ew-andrew-sus) [reply config/default/views.view.facilities_listing_blocks.yml:24]: Please compare the view result with the old version

File: config/default/views.view.find_a_campground_or_recreation_site.yml (hunk 4 above)
Comment (ew-andrew-sus) [reply config/default/views.view.find_a_campground_or_recreation_site.yml:14]: Please compare the view result with the old version

File: config/default/views.view.facilities_listing_blocks.yml (hunk 3 above)
Comment (ew-fedir) [reply config/default/views.view.facilities_listing_blocks.yml:24]: I reconfigured 'Facilities listing blocks' (facilities_listing_blocks) view based on D7 configuration.

File: config/default/views.view.find_a_campground_or_recreation_site.yml (hunk 4 above)
Comment (ew-fedir) [reply config/default/views.view.find_a_campground_or_recreation_site.yml:14]: I reconfigured 'Find a campground or recreation site' (find_a_campground_or_recreation_site) view based on D7 configuration.

**Review by ew-andrew-sus — APPROVED** [review]

**Pull request merged**